audb/core/dependencies.py

```diff
@@ -107,7 +107,7 @@ def archives(self) -> typing.List[str]:
             list of archives
 
         """
-        archives = [self.archive(file) for file in self.files]
+        archives = list(self._df.archive.values)
```

Comment:
```suggestion
        archives = self._df.archive.to_list()
```

I tested it and this way it's even faster: 0.00717380000060075 ->0.005856338000739925

Reply:
Btw: I also tested using `self._df.archive.unique()`, but this was much slower than using `set()`.

Reply:
Hehe, don't ask why, but good that you found an even faster solution.